### PR TITLE
Admin search user accounts

### PR DESCRIPTION
**Pull request by coreyd303**

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -4,12 +4,8 @@ class CoursesController < ApplicationController
...
 
   def index
-    @results = PgSearch.multisearch(params[:search]).includes(:searchable).map(&:searchable)
-    @courses = if params[:search].nil? || params[:search].empty?
-                 Course.includes(:lessons).all
-               else
-                 @results
-               end
+    results  = PgSearch.multisearch(params[:search]).includes(:searchable).map(&:searchable)
+    @courses = params[:search].blank? ? Course.includes(:lessons).all : results
```
Comment (ShepBook) [inline app/controllers/courses_controller.rb]: Cleaned up my code. :+1:

File: app/models/user.rb [hunk 2]
```diff
@@ -26,6 +26,15 @@
 #
 
 class User < ActiveRecord::Base
+  include PgSearch
+  # TODO: determine lockable? functionality and add to search
+  pg_search_scope :search_users, against: [:email],
```
Comment (ShepBook) [inline app/models/user.rb]: Love the postgres full text search.

**Comment by ShepBook** [comment]
Tweaked the visual display of users. The table was unhappy. I used a convention I started with the course list overview for normal users.

screen shot 2015-11-18 at 8 40 15 pm

**Comment by ShepBook** [comment]
Works great and looks good too now. :shipit:

**Comment by coreyd303** [comment]
Looks great!!

**Pull request merged**